fintorch/graph/layers/__init__.py

```
@@ -0,0 +1 @@
+from .beanconv import BeanAggregation, BeanAggregationSimple, BEANConv, BEANConvSimple
```

Ruff (0.8.2)

`.beanconv.BeanAggregation` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)